src/aiClients/useStateManager.ts

```diff
@@ -66,6 +66,7 @@ function useStateManager() {
     return managers;
   }, [initializing]);
   const [currentModel, setCurrentModel] = useState<Models | undefined>(initialModel);
+  const isCompact = true;
```

Comment:
I chose to remove the feature flag here as it was unclear we would ever turn compact off. Open to suggestions however.